heclib/heclib_c/src/Public/zinquire.c

```diff
@@ -136,11 +134,6 @@ long long zinquire(long long *ifltab, const char *request)
 		longNumber = 10000 * (ctemp[0] - '0') +
 		               100 * (ctemp[2] - (ctemp[2] < '[' ? '@' : '`')) +
 		                     (ctemp[3] - (ctemp[3] < '[' ? '@' : '`'));
```

Comment:
We should consider making a function such as  getVersionFromHeader(ifltab)  since this code is duplicated in  zinquireChar.c, and zinquire.c